### ISSUE TITLE
Error loading movie

### ISSUE THREAD

**Issue by HeronSantosCom** [markop159/KODI-Popcorn-Time#25]
First, great job!

It's happening this error when selecting any film, series or anime in the addon.

`17:45:24 T:140236880328448   FATAL: [plugin.video.kodipopcorntime] Traceback (most recent call last):
                                              File "/home/htpc/.kodi/addons/plugin.video.kodipopcorntime/addon.py", line 37, in <module>
                                                getattr(gui, params.pop('endpoint', 'index'))(params.pop('mediaType', '')).show(**params)
                                              File "/home/htpc/.kodi/addons/plugin.video.kodipopcorntime/resources/lib/kodipopcorntime/gui/player.py", line 79, in show
                                                item = self.getSelectedItem()
                                              File "/home/htpc/.kodi/addons/plugin.video.kodipopcorntime/resources/lib/kodipopcorntime/gui/player.py", line 51, in getSelectedItem
                                                "votes": xbmc.getInfoLabel('ListItem.Rating') and float(xbmc.getInfoLabel('ListItem.Votes')) or 0.0
`

As a temporary solution, only removed the words "float(xbmc.getInfoLabel('ListItem.Votes'))".

**Comment by markop159** [comment]
Hi, I am glad that you found the workaround this issue. But can you provide which OS and which version of KODI are you running.

**Comment by HeronSantosCom** [comment]
Sorry, here's the information:

OS/Distro: **Linux Mint 18 Sarah (GNU/Linux 4.4.0-21-generic x86_64)**
Kodi: **17.0-BETA5 Git:20161028-nogitfound Media Center Kodi**

**Comment by HeronSantosCom** [comment]
@markop159 I'm now using another OS, but I believe the problem is the version of Krypton.
OS: **macOS Sierra (10.12.1)**
Kodi: **Kodi 17.0-BETA6 Git: 20161112-ae37dd6**

**Comment by markop159** [comment]
In case of macOS Sierra there is a problem with firewall on the new version of macOS. I had the same problem when I updated from el capitan. If you disable firewall in macOS it will start working again.

**Comment by HeronSantosCom** [comment]
I checked here, but I thought it was strange because the Firewall is Off.

I believe that is not my network, because it didn't have this change. And when I was using the Kodi with Linux, I didn't have these problems.

In the previous installation, before you try the addon, I made some changes per recommendations of other addons. I'll check here and return if I had success.

Sorry for my bad English.

**Comment by HeronSantosCom** [comment]
@markop159 I did what I did in the old installation on another computer.
I had installed the Quasar, Pulsar, Plexus addon and then uninstalled it.
But without success.
I also had set some settings with the Easy Advanced Settings, and nothing.
Finally, I set the value of the gateway and routing. But still no go.
Takes too long to load a video in the addon, something around for 10 min.
What does not happen with torrents of YIFI via Transmission.
